src/components/authors/manageAuthorPage.js

```diff
@@ -74,20 +65,25 @@ var ManageAuthorPage = React.createClass({
 			AuthorActions.createAuthor(this.state.author);
 		}
 		
-		this.setState({dirty: false});
+		this.setState({dirty: false, redirect: true});

```

Comment:
Using the `Redirect` component below instead of `this.transitionTo`. `Redirect` references `this.state.redirect` to determine if the redirect should fire.